test/integration/acceptance/query_permission_test_base.cpp

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "query_permission_test_base.hpp"
+
+QueryPermissionTestBase::QueryPermissionTestBase(
+    const interface::RolePermissionSet &permission_to_query_myself,
+    const interface::RolePermissionSet &permission_to_query_my_domain,
+    const interface::RolePermissionSet &permission_to_query_everyone)
+    : kPermissionToQueryMyself(permission_to_query_myself),
+      kPermissionToQueryMyDomain(permission_to_query_my_domain),
+      kPermissionToQueryEveryone(permission_to_query_everyone) {}
+
+IntegrationTestFramework &QueryPermissionTestBase::prepareState(
+    AcceptanceFixture *fixture,
+    const interface::RolePermissionSet &spectator_permissions /*= {}*/,
```

Comment:
Why that many commented out `/*= {}*/`?

Reply:
it seems convenient to see the default values for arguments.

Reply:
Ah, I've got it. It's kind of a tip which cue about the header default values? If so that might cause inconsistency at the future in case someone would like to change thing at only one place

Reply:
And yeah, this kind of information (default args) is kinda useless to the implementation, I think

Reply:
removed these comments